docs/guides/permissions/overview.mdx

```diff
@@ -0,0 +1,214 @@
+
+### Defining Namespaces
+
+Relation tuples reside in [namespaces](/keto/concepts/namespaces.mdx). To create relation tuples, we first have to define their namespace.
+
+````mdx-code-block
+import CodeBlock from '@theme/CodeBlock'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="cloud" label="Ory CLI" default>
+
+```shell
+ory patch permission-config <your-project-id> \
+  --add '/namespaces/-={"id": 0, "name": "resources"}' \
+  --add '/namespaces/-={"id": 1, "name": "groups"}'
+```
+
+  </TabItem>
+  <TabItem value="self-hosted" label="Self-Hosted Ory Keto Config" default>
+
+```yaml title=path/to/keto.yml
+namespaces:
+  - id: 0
+    name: resources
+  - id: 1
+    name: groups
+```
+
+  </TabItem>
+</Tabs>
+````
+
+### Creating Relation Tuples
+
+Now we can use the [relation tuple APIs](/reference/api.mdx#operation/createRelationTuple) to create a relation tuple.
+
+````mdx-code-block
+<Tabs>
+  <TabItem value="cloud" label="Ory Cloud" default>
+
+```shell
+ORY_PAT="<your ory personal access token from the console>"
+ORY_SDK_URL="<your ory project sdk url from the console>"
+
+echo '[
+  {
+    "action": "insert",
+    "relation_tuple": {
+      "subject_id": "user1",
+      "relation": "member",
+      "namespace": "groups",
+      "object": "group1"
+    }
+  }, {
+    "action": "insert",
+    "relation_tuple": {
+      "subject_set": {
+        "relation": "member",
+        "namespace": "groups",
+        "object": "group1"
+      },
+      "relation": "owner",
+      "namespace": "resources",
+      "object": "file1"
+    }
+  }
+]' > relationtuple.json
+
+curl -X PATCH \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $ORY_PAT" \
+  -d @relationtuple.json \
+  "$ORY_SDK_URL/admin/relation-tuples"
+```
+
+  </TabItem>
+  <TabItem value="self-hosted" label="Self-Hosted Ory Keto" default>
+
+```shell
+KETO_WRITE_URL="<your keto write url>"
+
+echo '[
+  {
+    "action": "insert"
+    "relation_tuple": {
+      "subject_id": "user1",
+      "relation": "member",
+      "namespace": "groups",
+      "object": "group1"
+    }
+  }, {
+    "action": "insert"
+    "relation_tuple": {
+      "subject_set": {
+        "relation": "member",
+        "namespace": "groups",
+        "object": "group1"
+      },
+      "relation": "owner",
+      "namespace": "resources",
+      "object": "file1"
+    }
+  }
+]' > relationtuple.json
+
+curl -X PUT \
+  -H "Content-Type: application/json" \
+  -d @relationtuple.json \
+  "$KETO_WRITE_URL/admin/relation-tuples"
+```
+
+  </TabItem>
+</Tabs>
+````
+
+### Checking Permissions
+
+Permission checks are requests that check whether a subject has a relation to an object.
+This relation can either be directly defined in the database, or indirectly inferred through transitive relations.
```

Comment:
> or indirectly inferred through transitive relations

It's a lot of sophisticated words clustered together which might make understanding this sentence difficult for people of different levels of language proficiency. Could you try to rewrite this sentence to make it simpler? It's better to make it longer or explain something than write content that's hard to understand.